docs/dev/install.rst

```diff
@@ -267,6 +278,50 @@ you have to follow these steps:
 
 Once this is done, you should be able to trigger a new build on that project and it should succeed.
 
+
+Network connection failures
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After starting up your services with ``inv docker.up``, you might not be able to connect to the ``web`` instance on ``http://devthedocs.org:80``. Firstly, check all the outputs from Docker Compose to see that one of the instances hasn't crashed for a particular reason. If everything is up and running, it's likely related to a Docker network issue.
```

Comment:
Remember to use semantic break lines when writing documentation / comments. There are some paragraph that should be reviewed in this file.